app/controllers/admin/api/cms/templates_controller.rb

```diff
@@ -55,7 +56,7 @@ def index
   ##=  }
   ##=
   def create
-    type = cms_template_params.delete('type')
+    type = params.require('type')
```

Comment:
@akostadinov Do you remember why did you change it like this? It doesn't make sense to me

Reply:
probably some kind of mistake on my part and no second thought about it

I can fix it, or do you work on a PR that already fixes it?

Reply:
No need, already pushed the fix for it